webpack.config.js

```diff
@@ -19,7 +19,7 @@ let DEBUG_WEBPACK = !!process.env.DEBUG_WEBPACK;
 
 let config = dev.getDefaultWebpackConfig({
     projectRoot: __dirname,
-    verbosity: DEBUG_WEBPACK ? 'normal' : 'debug',
+    verbosity: DEBUG_WEBPACK ? 'debug' : 'normal',
```

Comment:
Is this the only extension where the flag is flipped? 
Also, I don't see any logic accounting for the flag being flipped

Reply:
It's already been fixed in a couple others.  I don't understand the second question.

Reply:
I didn't realize this was part of a config file. Scratch the second question